scripts/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -44,60 +45,62 @@ def create_new_calibration(self,
         """
         # vanadium corrections workspaces not used at this stage, but ensure they exist and create if not
         vanadium_corrections.fetch_correction_workspaces(vanadium_path, instrument, rb_num=rb_num)
-        ceria_workspace = path_handling.load_workspace(ceria_path)
         full_calib_path = get_setting(output_settings.INTERFACES_SETTINGS_GROUP,
```

Comment:
It seems like these ~7 lines relating to full_calib_path could also be moved inside the `if not only_update_vanadium` statement? I guess they also result in a workspace being loaded as well as populating the full_calib_path variable so we also need to consider if that workspace being present is an important part of the "Only update Vanadium" process. It seems like it's not to me because the user will always have either created\loaded a calibration prior to "Only update Vanadium"